src/Apps/__stories__/ArtworkApp.story.tsx

```diff
@@ -24,7 +24,7 @@ storiesOf("Apps/Artwork Page", module)
     return (
       <MockRouter
         routes={artworkRoutes}
-        initialRoute="/artwork2/david-hockney-diptychon-3"
+        initialRoute="/artwork2/berenice-abbott-nightview-new-york-1934"
```

Comment:
This is a bit pointless. All this auctions are going to be 'closed auctions' in a matter of several days:) Wish we had some year running auctions so that we can reliably see it in storybooks.